maven-shared-utils/src/main/java/org/apache/maven/shared/utils/logging/Style.java

```diff
@@ -21,48 +21,47 @@
 
 import org.fusesource.jansi.Ansi;
 import org.fusesource.jansi.Ansi.Attribute;
-import org.fusesource.jansi.Ansi.Color;
 
-import static org.fusesource.jansi.Ansi.Attribute.INTENSITY_BOLD;
+import java.lang.reflect.Method;
 
-import java.util.Locale;
+import static org.fusesource.jansi.Ansi.Attribute.INTENSITY_BOLD;
 
 /**
- * Configurable message styles.
+ *
  */
 enum Style
 {
 
-    DEBUG(   "bold,cyan"   ),
-    INFO(    "bold,blue"   ),
-    WARNING( "bold,yellow" ),
-    ERROR(   "bold,red"    ),
-    SUCCESS( "bold,green"  ),
-    FAILURE( "bold,red"    ),
+    DEBUG(   "bold,fgCyan"   ),
+    INFO(    "bold,fgBlue"   ),
+    WARNING( "bold,fgYellow" ),
+    ERROR(   "bold,fgRed"    ),
+    SUCCESS( "bold,fgGreen"  ),
+    FAILURE( "bold,fgRed"    ),
     STRONG(  "bold"        ),
-    MOJO(    "green"       ),
-    PROJECT( "cyan"        );
+    MOJO(        "fgGreen"   ),
+    PROJECT(     "fgCyan"    );
```

Comment:
Maybe not a good idea to be as coupled as Ansi library method names for configuration purpose.